apps/total-typescript/src/pages/thanks/purchase.tsx

```diff
@@ -110,88 +98,205 @@ const ThanksVerify: React.FC<
+        Your purchase is for <strong>{seatsPurchased}</strong> seat
+        {seatsPurchased > 1 && 's'}. You can always add more seats later when
+        your team grows.
+      </h1>
+      <h2 className="mx-auto max-w-lg py-5 font-text text-3xl font-bold lg:text-4xl">
+        Please check your inbox for a login link that just got sent.
+      </h2>
+      <code className="mb-10 mt-5 flex items-center justify-center gap-2 rounded-lg bg-cyan-500/10 px-4 py-3 font-sans text-base font-medium text-white shadow-xl sm:text-lg">
+        <MailIcon className="h-5 w-5 text-cyan-200" aria-hidden="true" />{' '}
+        <span className="text-cyan-200">Email sent to:</span> {email}
+      </code>
+      <p className="mx-auto max-w-xl pt-5 text-sm text-gray-200 sm:text-base sm:leading-relaxed">
+        As a final step to access the course you need to check your inbox (
+        <strong>{email}</strong>) where you will find an email from{' '}
+        <strong>{process.env.NEXT_PUBLIC_SUPPORT_EMAIL}</strong> with a link to
+        access your purchase and start learning.
+      </p>
+      <InlineTeamInvite bulkCouponId={bulkCouponId} />
+    </>
+  )
+}
+
+const ExistingBulkCouponPostPurchasePage: React.FC<
+  React.PropsWithChildren<{
+    product: SanityProduct
+    seatsPurchased: number
+    bulkCouponId: string
+  }>
+> = ({product, seatsPurchased, bulkCouponId}) => {
+  return (
+    <>
+      <h1 className="max-w-md text-lg font-medium text-cyan-100 sm:text-lg">
+        Thank you for purchasing more seats for{' '}
+        {product?.name || process.env.NEXT_PUBLIC_SITE_TITLE}!{' '}
+        <br className="hidden sm:block" />
+        Your purchase is for <strong>{seatsPurchased}</strong> additional seat
+        {seatsPurchased > 1 && 's'}.
+      </h1>
+      <InlineTeamInvite bulkCouponId={bulkCouponId} />
+    </>
+  )
+}
+
+const IndividualToBulkPostPurchasePage: React.FC<
+  React.PropsWithChildren<{
+    product: SanityProduct
+    seatsPurchased: number
+    bulkCouponId: string
+  }>
+> = ({product, seatsPurchased, bulkCouponId}) => {
+  return (
+    <>
+      <h1 className="max-w-md text-lg font-medium text-cyan-100 sm:text-lg">
+        Thank you for purchasing more seats for{' '}
+        {product?.name || process.env.NEXT_PUBLIC_SITE_TITLE}!{' '}
+        <br className="hidden sm:block" />
+        Your purchase is for <strong>{seatsPurchased}</strong> additional seat
+        {seatsPurchased > 1 && 's'}. You can always add more seats later when
+        your team grows.
+      </h1>
+      <InlineTeamInvite bulkCouponId={bulkCouponId} />
+    </>
+  )
+}
+
+const ThanksVerify: React.FC<
+  React.PropsWithChildren<{
+    email: string
+    seatsPurchased: number
+    purchaseType: PurchaseType
+    bulkCouponId: string
+    product: SanityProduct
+  }>
+> = ({email, seatsPurchased, purchaseType, bulkCouponId, product}) => {
+  let postPurchasePage = null
+
+  switch (purchaseType) {
+    case NEW_INDIVIDUAL_PURCHASE:
```

Comment:
I'm switching on each post-purchase scenario to build a component that renders the copy and CTA relevant to that scenario. They all look pretty similar at the moment, but this is now a basis to better hone the messaging and CTA on each.